src/main/java/com/criticalsoftware/Product.java

```diff
@@ -1,23 +1,32 @@
 package com.criticalsoftware;
+
 import org.bson.types.ObjectId;
-import java.awt.image.BufferedImage;
 
 public class Product {
 
     private ObjectId id;
```

Comment:
As the id field is already inherited from the parent class (PanacheMongoEntity), it doesn't need to be explicitly declared in this class.

Reply:
Done.